sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -327,8 +315,8 @@
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.getSettingWithResponse#ConfigurationSetting-OffsetDateTime-boolean}
      *
      * @param setting The setting to retrieve.
-     * @param asOfDateTime To access a past state of the configuration setting, or optionally, null if a setting with
-     * {@code asOfDateTime} is desired.
+     * @param acceptDateTime To access a past state of the configuration setting, or optionally, null if a setting with
```

Comment:
I don't get this doc. "optionally null if a setting with acceptDateTime is desired"? If it's null, what time am I looking for exactly?

Reply:
the acceptDateTime is optional and pass null if you don't want to consider acceptDateTime